modules/ztp-install-openshift-pipelines.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="install-the-openshift-pipelines-operator_{context}"]
+= Installing the {product-title} Pipelines Operator
+
+Follow this guidance to install the OpenShift Pipelines Operator that is used to run the pipeline.
+
+.Prerequisites
+
+* An {product-title} hub cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install `git`. For guidance on installing `git`, see link:https://github.com/git-guides/install-git[Install Git].
+
+.Procedure
+
+. Export the `KUBECONFIG` environment variable.
```

Comment:
Most other places you use `:` to go into a command (and that's consistent with what we try to do), so I'd suggest updating here and in the next step. Might be other places in the PR if you could do a quick check